views/email/email_partials/report_what_to_do_next.hbs

```diff
@@ -7,34 +7,57 @@
     
     <tr>
-      <td colspan="2" valign="top" style="padding-top: 10px;">
+      <td valign="top" width="60" style="width: 60px; padding-top: 15px;">
+        <img width="45" src="{{ SERVER_URL }}/img/email_images/wtd-1.png" alt="Number 1" />
+      </td>
+      <td valign="top" style="padding-top: 10px;">
         <p class="body-copy" style="color: #333333; font-weight: 400; font-size: 16px; padding: 0px; margin: 0px; ">
-          Change your passwords, even for old accounts
+          {{fluentFormat supportedLocales "what-to-do-subhead-1"}}
         </p>
         <p class="body-copy" style="color: #333333; font-weight: 300; font-size: 14px; padding: 3px 0px 10px 0px; margin-top: 0px; ">
-          If you can’t log in, contact the website to ask how you can recover or shut down the account. See an account you don’t recognize? The site may have changed names or someone may have created an account for you.
+          {{fluentFormat supportedLocales "what-to-do-blurb-1"}}
         </p>
       </td>
     </tr>
 
     <tr>
-      <td colspan="2" valign="top" style="padding-top: 10px;">
+      <td valign="top" width="60" style="width: 60px; padding-top: 15px;">
+        <img width="45" src="{{ SERVER_URL }}/img/email_images/wtd-2.png" alt="Number 1" />
+      </td>
+      <td valign="top" style="padding-top: 10px;">
         <p class="body-copy" style="color: #333333; font-weight: 400; font-size: 16px; padding: 0px; margin: 0px; ">
-          If you reuse an exposed password, change it
+          {{fluentFormat supportedLocales "what-to-do-subhead-2"}}
         </p>
         <p class="body-copy" style="color: #333333; font-weight: 300; font-size: 14px; padding: 3px 0px 10px 0px; margin-top: 0px; ">
-          Hackers may try to reuse your exposed password to get into other accounts. Create a different password for each website, especially for your bank account, email and other websites where you save personal information.
+          {{fluentFormat supportedLocales "what-to-do-blurb-2"}}
         </p>
       </td>
     </tr>
 
     <tr>
-      <td colspan="2" valign="top" style="padding-top: 10px;">
+      <td valign="top" width="60" style="width: 60px; padding-top: 15px;">
+        <img width="45" src="{{ SERVER_URL }}/img/email_images/wtd-3.png" alt="Number 1" />
+      </td>
+      <td valign="top" style="padding-top: 10px;">
+        <p class="body-copy" style="color: #333333; font-weight: 400; font-size: 16px; padding: 0px; margin: 0px; ">
+          {{fluentFormat supportedLocales "what-to-do-subhead-3"}}
+        </p>
+        <p class="body-copy" style="color: #333333; font-weight: 300; font-size: 14px; padding: 3px 0px 10px 0px; margin-top: 0px; ">
+          {{fluentFormat supportedLocales "what-to-do-blurb-3"}}
+        </p>
+      </td>
+    </tr>
+
+    <tr>
+      <td valign="top" width="60" style="width: 60px; padding-top: 15px;">
+        <img width="45" src="{{ SERVER_URL }}/img/email_images/wtd-4.png" alt="Number 1" />
```

Comment:
“Number 4”

Reply:
Micro-nit: now that these are localized, they feel repetitive. Not sure if they’d be a candidate for a partial. Only thing that seems to change is:
- “wtd-4.png”
- alt=“Number 4”
- “what-to-do-subhead-4” (L57 below)
- “what-to-do-blurb-4” (L60 below)